src/components/ArticleActions.vue

```diff
@@ -81,10 +81,13 @@ export default {
         username: this.profile.username
       });
     },
-    deleteArticle(slug) {
-      this.$store.dispatch(ARTICLE_DELETE, slug).then(() => {
+    async deleteArticle(slug) {
+      try {
+        await this.$store.dispatch(ARTICLE_DELETE, slug);
         this.$router.push("/");
-      });
+      } catch (err) {
+        console.log(err);
```

Comment:
Let's rather log with the proper error message `console.error()`.

You can find more information on MDN about it: https://developer.mozilla.org/en-US/docs/Web/API/Console/error

Reply:
Done.